src/AI/WallyAI.java

```diff
@@ -97,11 +97,12 @@ private void init(GameMap map)
             if( vroom.canTraverse(terrain) )
             {
               validTiles++;
-              totalCosts = vroom.getMoveCost(terrain);
+              totalCosts += vroom.getMoveCost(terrain);
```

Comment:
Ah! Yes it should.

Reply:
You can probably find the moment I discovered that, by my lamentations on Discord